components/mediation-ui/mediators-ui/org.wso2.carbon.mediator.enrich.ui/src/main/resources/web/enrich-mediator/edit-mediator.jsp

```diff
@@ -60,11 +61,13 @@
     } else if (enrichMediator.getSourceType().equals("custom")) {
         sourceType = CUSTOM;
         if (enrichMediator.getSourceExpression() != null) {
-            SynapseXPath sourceXPath = enrichMediator.getSourceExpression();
-            NameSpacesRegistrar nameSpacesRegistrar = NameSpacesRegistrar.getInstance();
-            if (sourceXPath != null) {
-                nameSpacesRegistrar.registerNameSpaces(sourceXPath, "mediator.enrich.source.val_ex", session);
-                sourceValue = sourceXPath.toString();
+            SynapsePath sourcePath = enrichMediator.getSourceExpression();
+            if (sourcePath != null) {
```

Comment:
Don't we need to have a log if sourcePath is null?

Reply:
+1 added